lib/device.js

```diff
@@ -260,6 +264,11 @@ class Device extends EventEmitter {
 		this._testAppBinFile = null;
 	}
 
+	async getUsbDevice() {
+		await this._open();
+		return this._usbDev;
```

Comment:
`_open()` can fail, and `_usbDev` can be `null`. I'm guessing that will trigger the test failure in this case, which should be acceptable?

Reply:
Yes, this should be fine.